delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/IOUtils.scala

```diff
@@ -17,6 +17,15 @@ trait IOUtils {
     clock.realTime(TimeUnit.MILLISECONDS).map(Instant.ofEpochMilli)
 }
 
+object IOUtils extends IOUtils
```

Comment:
`IOUtils` is gonna be replaced with `IOInstant` progressively

Reply:
Yes, but `IOInstant.get` is less readable than something like `now` or `Clock.now` where the intention is clearer

If you imported `get` then you would have to rename it everywhere, so calling it `now` instead of get helps